### PR TITLE
Upgrade Flow

### PR DESCRIPTION
**Pull request by zpao**
Some of this was done in a codemod by @bhosmer.

Since we aren't syncing all the RN stuff in, there are more failures there unrelated to ReactElement but for other reasons. We should clean those up too since flow should actually be failing right now.

### CODE REVIEW

**Comment by spicyj** [comment]
👍

**Comment by zpao** [comment]
Fixed the rest of it too and just combined with other `<any>` args (Basil only fixed the things that were synced at that point).

File: src/renderers/native/ReactNativeAttributePayload.js [hunk 1]
```diff
@@ -394,9 +394,9 @@ function diffProperties(
       // default: fallthrough case when nested properties are defined
       removedKeys = null;
       removedKeyCount = 0;
-      // $FlowFixMe - We think that attributeConfig is not
-      // CustomAttributeConfiguration at this point so we assume
-      // it must be AttributeConfiguration.
```
Comment (zpao) [inline src/renderers/native/ReactNativeAttributePayload.js]: @gabelevi - Not sure if you or somebody else would want to make this multi-line format work. I would have expected this to work but apparently it doesn't just apply it to the next real code line and presumably just applies it to the next line which in this case was a comment.

**Comment by facebook-github-bot-2** [comment]
@zpao updated the pull request.

**Pull request merged**